core/src/main/java/zipkin2/reporter/BytesMessageEncoder.java

```diff
@@ -14,6 +14,7 @@
 package zipkin2.reporter;
 
 import java.util.List;
+import org.omg.PortableServer.THREAD_POLICY_ID;
```

Comment:
😱 OMG
I mean, is this import a mistake? 😝

Reply:
good catch. unused by needed to be removed! 11b612d5b54d6fd57f52b2b94e622a77768f3cbf